light/client.go

```diff
@@ -658,6 +713,34 @@ func (c *Client) verifySequential(
 			}
 		}
 
+		// 2.1) Verify that the data behind the block data is actually available.
+		if c.verificationMode == dataAvailabilitySampling {
+			start := time.Now()
+			// TODO: decide how to handle this case:
+			// https://github.com/lazyledger/lazyledger-core/issues/319
+			numRows := len(interimBlock.DataAvailabilityHeader.RowsRoots)
+			numSamples := min(c.numSamples, numRows*numRows)
+			c.logger.Info("Starting DAS sampling", "height", height, "numSamples", numSamples, "squareWidth", numRows)
```

Comment:
"DAS sampling" means "data availability sampling sampling" :P

Reply:
Yeah, will change that soon 😅

Reply:
https://github.com/lazyledger/lazyledger-core/pull/323/commits/686525c9e42db4588873182f3eafd6812194a36c